vagrant/Vagrantfile

```diff
@@ -107,8 +55,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         # Default value: false
         target.ssh.forward_agent = true
 
-        target.vm.provision "file", source: "./upgrade_packages.sh", destination: "upgrade_packages.sh"
-        target.vm.provision "file", source: "./blacklist_mongo.txt", destination: "blacklist_mongo.txt"
+        #target.vm.provision "file", source: "./upgrade_packages.sh", destination: "upgrade_packages.sh"
+        #target.vm.provision "file", source: "./blacklist_mongo.txt", destination: "blacklist_mongo.txt"
```

Comment:
It's used for install latest mongo which not in the version blacklist.
But we restrict the mongo version now, so this script is left off.